WooCommerce/Classes/ViewRelated/Orders/Simple Payments/Amount/SimplePaymentsAmountViewModel.swift

```diff
@@ -146,30 +165,23 @@ final class SimplePaymentsAmountViewModel: ObservableObject {
 // MARK: Helpers
 private extension SimplePaymentsAmountViewModel {
 
-    /// Formats a received value by making sure the `$` symbol is present and trimming content to two decimal places.
-    /// TODO: Update to support multiple currencies
+    /// Formats a received value by sanitizing the input and trimming content to two decimal places.
     ///
-    func formatAmount(_ amount: String) -> String {
+    func sanitizeAmount(_ amount: String) -> String {
         guard amount.isNotEmpty else { return amount }
 
         let deviceDecimalSeparator = userLocale.decimalSeparator ?? "."
         let storeDecimalSeparator = storeCurrencySettings.decimalSeparator
 
         // Removes any unwanted character & makes sure to use the store decimal separator
-        var formattedAmount = amount
+        let sanitized = amount
             .replacingOccurrences(of: deviceDecimalSeparator, with: storeDecimalSeparator)
-            .filter { $0.isNumber || $0.isCurrencySymbol || "\($0)" == storeDecimalSeparator }
-
-        // Prepend the store currency symbol if needed.
-        // TODO: We are appending the currency symbol always to the left, we should use `CurrencyFormatter` when releasing to more countries.
-        if !formattedAmount.hasPrefix(storeCurrencySymbol) {
-            formattedAmount.insert(contentsOf: storeCurrencySymbol, at: formattedAmount.startIndex)
-        }
+            .filter { $0.isNumber || "\($0)" == storeDecimalSeparator }
 
         // Trim to two decimals & remove any extra "."
```

Comment:
Since we're supporting any currency, we also need to change how we trim the amount because stores can change their currency settings to support more than 2 decimal places. In that case, the placeholder is formatted with the expected number of decimal places but we only accept two.